pyron-engine/src/main/scala/com/cloudentity/pyron/Application.scala

```diff
@@ -54,7 +54,7 @@ class Application extends VertxBootstrap with FutureConversions with ScalaSyntax
   override def deployServer(): VxFuture[String] = {
     for {
       _          <- deployApiHandlers(appConf)
-      _          <- deployServerInstance(appConf)
+      _          <- deployServerInstances(appConf, getServerVerticlesNum(appConf), Future.successful(""))
```

Comment:
Please revert accidentally checked-in changes in this file. Compare with current dev branch.